.github/label-change.yml

```diff
@@ -31,3 +31,28 @@ ci:
   - ".github/**"
   - "dockers/**"
   - ".pre-commit-config.yml"
+
+docs:
+  - ".github/workflows/docs-build.yml"
```

Comment:
Touching CI files is not generally about docs

```suggestion
```

Reply:
but it is affecting docs